### ISSUE TITLE
Create page error: The given key was not present in the dictionary

### ISSUE THREAD

**Issue by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#209]
I tried to install the 86 module on a 32 bit compatible app pool and the 64 on a 64 appl pool but both throw an error if you want to create a Vanjaro page or migrate an existing page:

Server Error in '/' Application.
The given key was not present in the dictionary.
Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code.

Exception Details: System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.

Source Error:

An unhandled exception was generated during the execution of the current web request. Information regarding the origin and location of the exception can be identified using the exception stack trace below.

Stack Trace:

[KeyNotFoundException: The given key was not present in the dictionary.]
   System.Collections.Generic.Dictionary`2.get_Item(TKey key) +12834479
   Vanjaro.Migrate.Components.HttpModule.context_PreRequestHandlerExecute(Object sender, EventArgs e) +742
   System.Web.SyncEventExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute() +201
   System.Web.HttpApplication.ExecuteStepImpl(IExecutionStep step) +132
   System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) +73

Version Information: Microsoft .NET Framework Version:4.0.30319; ASP.NET Version:4.8.4210.0

**Comment by meetmandeep** [comment]
@Tychodewaard I'm unable to duplicate this on a new DNN 9.7.2 installation with Vanjaro 0.5.2. 

Did you upgrade from an existing version? If so, can you confirm what version did you upgrade from? 

DNN 9.7.0 -> 9.7.2 ? 
Vanjaro 0.5.1 -> 0.5.2 ?

**Comment by Tychodewaard** [comment]
This is a fresh 972 at managed.com

Outlook voor iOS<https://aka.ms/o0ukef> downloaden
________________________________
Van: Mandeep Singh <notifications@github.com>
Verzonden: Wednesday, October 14, 2020 8:17:30 PM
Aan: vanjarosoftware/Vanjaro.Platform <Vanjaro.Platform@noreply.github.com>
CC: Tycho de Waard <info@tjeps.com>; Mention <mention@noreply.github.com>
Onderwerp: Re: [vanjarosoftware/Vanjaro.Platform] Create page error: The given key was not present in the dictionary (#209)


@Tychodewaard<https://github.com/Tychodewaard> I'm unable to duplicate this on a new DNN 9.7.2 installation with Vanjaro 0.5.2.

Did you upgrade from an existing version? If so, can you confirm what version did you upgrade from?

DNN 9.7.0 -> 9.7.2 ?
Vanjaro 0.5.1 -> 0.5.2 ?

**Comment by Tychodewaard** [comment]
Just installed again which went according to plan.
I did a couple of things differently:
- Set app pool on 64 bit before installing DNN 9.7.2 (previous time I installed DNN and then changed the appl pool which was 32 bit by default)
- I set the url that I use as primary (previous time it was not primary)
- I installed with nothing but DNN. (previous time there was AppBuilder and EasyDNNMaps already installed.
